### ISSUE TITLE
unable to install/run stormloader.application

### ISSUE THREAD

**Issue by SolomonCM** [Lewinator56/StormLoader#15]
i am getting an error when i try to run stormloader.application. i have attached a copy of the log. it appears some files are missing?

[S0ISK17J.log](https://github.com/Lewinator56/StormLoader/files/11195477/S0ISK17J.log)

**Comment by Lewinator56** [comment]
Can you attach the log from stormloader, the log you have attached is not the one generated by stormloader

**Comment by BoppinAfro** [comment]
Hello I just got this and tried it today I am having the same issues and I am not folder savvy still trying to self learn develop so I will send to files hope they are the logs you are looking for
[log.txt](https://github.com/Lewinator56/StormLoader/files/11872504/log.txt)
[error details straight from Cannot Start Application.txt](https://github.com/Lewinator56/StormLoader/files/11872505/error.details.straight.from.Cannot.Start.Application.txt)

**Comment by Lewinator56** [comment]
> Hello I just got this and tried it today I am having the same issues and I am not folder savvy still trying to self learn develop so I will send to files hope they are the logs you are looking for [log.txt](https://github.com/Lewinator56/StormLoader/files/11872504/log.txt) [error details straight from Cannot Start Application.txt](https://github.com/Lewinator56/StormLoader/files/11872505/error.details.straight.from.Cannot.Start.Application.txt)

is seems to not be able to load a file before the application actually starts - the error isnt logged by stormloader so its a windows error. have you extracted stormloader before running it_

**Comment by BoppinAfro** [comment]
Yeah I have done the guide here and what discord said to the point I dont think its an issue on my end as its not just me other communities have complained this mod is not working for them either so I know for sure not just my laptop having the issue

**Comment by Lewinator56** [comment]
Every time I test StormLoader I cannot replicate the issue - what language is your system set to? do you have admin rights? does windows have appropriate permissions to access the files in the folder you have stormloader in?

**Comment by Lewinator56** [comment]
> Yeah I have done the guide here and what discord said to the point I dont think its an issue on my end as its not just me other communities have complained this mod is not working for them either so I know for sure not just my laptop having the issue

This appears to be an issue with net framework loading data files, the error occurs BEFORE stormloader starts loading any of its internal stuff - the error is specifically a windows error: System.Deployment.Application.DeploymentDownloadException as part of the JIT compiler for .Net, can you try running 'sfc /scannow' in an administrator command prompt

**Comment by Lewinator56** [comment]
Can I check too that you are running the prebuilt application? your error relates to the clickonce application deployment, stormloader DOESNT use this, so unless you are building yourself and deploying through clickonce, i cant see how this error can occur

**Comment by BoppinAfro** [comment]
I dunno I give up I just did manual just going to suck when issues come up this loader would prevent and I dont get that protection I have never had any issue with anything else but nothing else also had a click once application and not so I dont really know it says its missing specific files every time I have had this error

**Comment by Lewinator56** [comment]
> I dunno I give up I just did manual just going to suck when issues come up this loader would prevent and I dont get that protection I have never had any issue with anything else but nothing else also had a click once application and not so I dont really know it says its missing specific files every time I have had this error

the best thing i can recommend is you pull the source code and try building it yourself - as i say I cant replicate this issue on any system I have access to (thats 6 different systems). Its pretty easy, you just need to clone the repo in visual studio and click build.

**Comment by Pomfe** [comment]
Hey there. Having an issue preventing me from loading Stormloader, it keeps saying "not enough quota is available to process this command." I have attached my log
[log.txt](https://github.com/Lewinator56/StormLoader/files/11927703/log.txt)

**Comment by Lewinator56** [comment]
> Hey there. Having an issue preventing me from loading Stormloader, it keeps saying "not enough quota is available to process this command." I have attached my log [log.txt](https://github.com/Lewinator56/StormLoader/files/11927703/log.txt)

This is typically associated with having your pagefile switched off. Check if you have the pagefile enabled under performance settings in windows.

**Comment by Pomfe** [comment]
![sshot-2](https://github.com/Lewinator56/StormLoader/assets/65171795/c3139464-a6fd-4ebc-99b3-4ae7d2a7b94d)
this is what I see, not sure if this is what you are talking about or not though

**Comment by Lewinator56** [comment]
> ![sshot-2](https://user-images.githubusercontent.com/65171795/250313408-c3139464-a6fd-4ebc-99b3-4ae7d2a7b94d.png) this is what I see, not sure if this is what you are talking about or not though

Try putting it on another drive, running it as admin or rebooting - this is specifically a WINDOWS issue and not anything to do with the code for stormloader.

**Comment by Pomfe** [comment]
thanks for the advice, I'll get back to you depending on how it goes!

**Comment by Pomfe** [comment]
Restarting seemed to do the job, thank you
